src/SerConAai.csproj

```diff
@@ -79,11 +79,15 @@
     </PackageReference>
     <PackageReference Include="Hjson" Version="3.0.0" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
-    <PackageReference Include="NLog" Version="5.0.0-beta11" />
+    <PackageReference Include="NLog" Version="4.5.4" />

```

Comment:
why you change back to 4.x.x ist this now also .netcore ready

Reply:
@konne 
The version 5.0.0 beta is no longer available. The max version is 4.5.4